### DISCUSSION TITLE
Question about Character Review Scheduling

### DISCUSSION THREAD

**Discussion by ChuckreimuHakunorris** [syt0r/Kanji-Dojo#78] [General]
Hi,

First of all, thank you for making this great app! I've been using it a lot to brush up on my kanji, I find the writing review especially useful since having to write the characters commits them to memory better than just reading, I find.

I have a question about the review scheduling, how does the logic for that work exactly? I assume it's something like the more times you get a character correct, the further apart the reviews are phased out? Is there a limit to this, I mean at some point does the program consider a character "learned" and stops scheduling it for review?

As a future feature suggestion, showing the learning progress info per character and providing some settings relating to this would be greatly appreciated.

**Comment by syt0r** [comment]
Hello. The interval since the last review until the next one is calculated like this: `1.1 * 1 day * repeats` where repeats is how many times user already wrote this character without a fail, it starts from 1 and resets whenever user marks it as a fail in the review summary screen

There are no limits or thresholds to stop scheduling

**Reply by ChuckreimuHakunorris** [reply]
Alright, thanks for the info!
